app/src/app/items/page.tsx

```
@@ -681,27 +989,49 @@ const Character: React.FC<CharacterProps> = (props) => {
               />
             )}
           </Modal2>
         )}
+        {/* Repair Item Selection Modal */}
+        <RepairItemSelectionModal
+          isOpen={isRepairModalOpen}
+          setIsOpen={setIsRepairModalOpen}
+          targetItem={useritem!}
+          repairItems={repairItems}
+          onSelectRepairItem={(repairItemId, targetItemId) => {
+            useRepairItem({
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)